spec/dummy/app/themes/default/assets/stylesheets/thredded.scss

```diff
@@ -1,15 +1,25 @@
+@import url(http://fonts.googleapis.com/css?family=Lato:400,900,400italic);
+
 @import "bourbon";
-@import "base/base";
+@import "base/grid-settings";
 @import "neat";
-@import "thredded/variables";
-@import "thredded/common";
-@import "thredded/layout";
-@import "thredded/nav";
-@import "thredded/sidebar";
-@import "thredded/messageboards";
-@import "thredded/topics/common";
-@import "thredded/topics/public";
-@import "thredded/topics/private";
-@import "thredded/posts";
-@import "thredded/drop_forms";
+@import "base/base";
+
 @import "vendor/chosen";
+
+@import "utilities/is-hidden";
+@import "utilities/is-visible";
+@import "utilities/is-in-the-background";
+
+@import "components/main-container";
+@import "components/user-navigation";
+@import "components/topic-navigation";
+@import "components/main-navigation";
+
+@import "components/messageboard";
+@import "components/drop-form";
+
+@import "components/main-section";
+
+@import "thredded/drop_forms";
+@import "thredded/topics/common";
```

Comment:
Let me be clear though - I love the way you broke all these out. Holy crap, so nice.